src/components/newMeasure/NewMeasure.test.tsx

```diff
@@ -2,12 +2,10 @@ import "@testing-library/jest-dom";
 // NOTE: jest-dom adds handy assertions to Jest and is recommended, but not required
 
 import * as React from "react";
-import { render, fireEvent, screen } from "@testing-library/react";
-import NewMeasure from "../NewMeasure/NewMeasure";
+import { render, screen } from "@testing-library/react";
+import NewMeasure from ".//NewMeasure";
```

Comment:
probably just ` ./NewMeasure` would be fine

Reply:
Yes, removing it